tests/Unit/Entity/Activity/PriceOptionTest.php

```diff
@@ -200,13 +204,26 @@ public function testGetRegistrations(): void
 
     public function testAddRegistration(): void
     {
-        /* @todo This test is incomplete. */
-        $this->markTestIncomplete();
+        $expected = new Registration();
+        $property = (new ReflectionClass(PriceOption::class))
+            ->getProperty('registrations');
+        $property->setAccessible(true);
+        $this->priceOption->addRegistration($expected);
+        $this->assertSame($expected, $property->getValue($this->priceOption)[0]);
     }
 
     public function testRemoveRegistration(): void
     {
-        /* @todo This test is incomplete. */
-        $this->markTestIncomplete();
+        $expected = new ArrayCollection();
+        $registration = new Registration();
+        $expected->add($registration);
+        $property = (new ReflectionClass(PriceOption::class))
+            ->getProperty('registrations');
+        $property->setAccessible(true);
+        $property->setValue($this->priceOption, $expected);
+        $this->assertSame($registration, $property->getValue($this->priceOption)[0]);
```

Comment:
On all the testRemove* methods added (for example here), this early assert is added. Why? To me, it feels like left in debugging code, as it's not testing the actual remove method, but instead it's testing whether the addition of the data has succeeded. I feel like that may be safely assumed given the fact that we're using reflection to populate the value. I'd like to see it removed everywhere.

Reply:
It wasn't test code per se, more just an extra check to make sure the start conditions are met. However I do agree that we should be able to just assume the reflectionclass does what it's supposed to, so it does not add anything. consider it removed.